homeassistant/components/history/websocket_api.py

```diff
@@ -130,6 +131,14 @@ async def ws_get_history_during_period(
         return
 
     entity_ids = msg.get("entity_ids")
+    if entity_ids is not None:
```

Comment:
We can change the schema to make it required here and than drop the None check as well since it's going to throw deeper in the stack if it's missing anyways

Reply:
Is the `entity_id` supposed to be required? Previously if you did not supply it you got the history data for all entities.

I ask because I replaced `vol.Optional("entity_ids"): [str],` with `vol.Required("entity_ids"): [str],` which causes the tests `test_history_during_period_bad_start_time` and `test_history_during_period_bad_end_time` to fail which might make it a breaking change and idk what to do about that, do I remove the failing test cases?

Reply:
We need to keep the tests for coverage but you can add entity ids to the test cases